spec/features/hq/user_is_creating_multiple_sponsors_spec.rb

```diff
@@ -16,18 +16,11 @@
   end
 end
 
+
 def an_agent_exists
   FactoryGirl.create :agent
 end
 
-def i_sign_in_as_admin
```

Comment:
I'm still confused: How are these method definitions shared between files? Can the 'user_is_creating...' feature run on its own?